packages/manager/modules/netapp/package.json

```diff
@@ -25,6 +25,7 @@
     "@ovh-ux/manager-core": "^11.0.0",
     "@ovh-ux/manager-models": "^1.11.1",
     "@ovh-ux/manager-ng-layout-helpers": "^2.0.0",
+    "@ovh-ux/manager-filters": "^0.1.0 || ^1.0.0",
```

Comment:
**issue(blocking):** If this package is required, then we have to import/use it in `package/manager/modules/netapp/*`. Correct me if I'm wrong, but it's not the case here.

If package named `@ovh-ux/manager-filters` is required as a `peerDependency` for the given module `@ovh-ux/manager-netapp`, we have to add it also as a `dependency` for the standalone application `@ovh-ux/manager-netapp-app`.

:bulb: **Tips:** It's highly recommended to use the standalone application to develop features/bugfixes for product section.

```sh
$ yarn workspace @ovh-ux/manager-netapp-app run start:dev
```

Reply:
Netapp is not part of dedicated univers ?
Because i see in dedicated and we have this deps.

Reply:
NetApp and Dedicated are two separated Apps but both are using same module `@ovh-ux/manager-netapp`.